### PR TITLE
Intial Implementation of Container in homepage

### PR DESCRIPTION
**Pull request by chandrakailash1993**
Implemented New API and Added Container Pattern

### CODE REVIEW

File: app/components/atoms/Title/tests/__snapshots__/Title.test.js.snap [hunk 1]
```diff
@@ -1,25 +1,41 @@
...
 
 exports[`Anchor Component should render correctly and match snapshot 1`] = `
-.c0 {
-  opacity: 1;
-}
-
-<Title
-  className="c0"
-  inheritedStyles=""
+<StyledComponent
```
Comment (vinodloha) [inline app/components/atoms/Title/tests/__snapshots__/Title.test.js.snap]: This change is not correct, use the vanilla export to test the component.

File: app/components/atoms/Button/tests/__snapshots__/Button.style.test.js.snap [hunk 2]
```diff
@@ -1,20 +1,32 @@
...
-  ariaLabel=""
-  className="c0"
-  disabled={false}
-  inheritedStyles=""
-  loader={false}
-  loaderPositionCenter={false}
-  primary={false}
-  secondary={false}
-  tabIndex=""
+<StyledComponent
```
Comment (vinodloha) [inline app/components/atoms/Button/tests/__snapshots__/Button.style.test.js.snap]: Here use ButtonVanilla

File: app/components/atoms/Button/tests/__snapshots__/Button.style.test.js.snap [hunk 3]
```diff
@@ -35,25 +47,36 @@ exports[`Button should render correctly 1`] = `
...
```
Comment (vinodloha) [inline app/components/atoms/Button/tests/__snapshots__/Button.style.test.js.snap]: Same use ButtonVanilla

File: app/components/atoms/Image/tests/__snapshots__/Image.style.test.js.snap [hunk 4]
```diff
@@ -1,9 +1,33 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`Image Component with styles should render correctly and match snapshots 1`] = `
-<Image
+<StyledComponent
```
Comment (vinodloha) [inline app/components/atoms/Image/tests/__snapshots__/Image.style.test.js.snap]: Same use ImageVanilla

File: app/components/atoms/ProgressCircle/tests/__snapshots__/ProgressCircle.style.test.js.snap [hunk 5]
```diff
@@ -1,35 +1,32 @@
...
-  stroke: #eeeeee;
-}
-
-.c0 text {
-  font-size: 1rem;
-}
-
-<ProgressRing
-  className="c0"
+<StyledComponent
```
Comment (vinodloha) [inline app/components/atoms/ProgressCircle/tests/__snapshots__/ProgressCircle.style.test.js.snap]: Here use ProgressRingVanilla

File: app/components/atoms/TextArea/tests/__snapshots__/TextArea.test.js.snap [hunk 6]
```diff
@@ -19,14 +19,35 @@ exports[`PaymentMethod component should render correctly 1`] = `
 `;
 
 exports[`PaymentMethodStyle component should render correctly 1`] = `
-.c0 {
-  opacity: 1;
-}
-
-<TextArea
-  className="c0"
+<StyledComponent
```
Comment (vinodloha) [inline app/components/atoms/TextArea/tests/__snapshots__/TextArea.test.js.snap]: We should have deleted this TextArea component from here and used the one form component lib

File: app/components/molecules/NavBar/tests/__snapshots__/NavBar.test.js.snap [hunk 7]
```diff
@@ -1,13 +1,33 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`<NavBar /> should render correctly 1`] = `
-.c0 {
-  opacity: 1;
-}
-
-<NavBar
-  className="c0"
+<StyledComponent
```
Comment (vinodloha) [inline app/components/molecules/NavBar/tests/__snapshots__/NavBar.test.js.snap]: Use NavBarVanilla

File: app/components/molecules/TopBanner/tests/__snapshots__/TopBanner.test.js.snap [hunk 8]
```diff
@@ -1,13 +1,33 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`<TopBanner /> should render correctly 1`] = `
-.c0 {
-  opacity: 1;
-}
-
-<TopBanner
-  className="c0"
+<StyledComponent
```
Comment (vinodloha) [inline app/components/molecules/TopBanner/tests/__snapshots__/TopBanner.test.js.snap]: Use TopBannerVanilla

File: app/components/atoms/Title/tests/__snapshots__/Title.test.js.snap (hunk 1 above)
Comment (chandrakailash1993) [reply app/components/atoms/Title/tests/__snapshots__/Title.test.js.snap]: Done

File: app/components/templates/HomePage/HomePage.js [hunk 9]
```diff
@@ -1,12 +1,15 @@
 // @flow
 import React, { memo } from 'react';
 import type { Node } from 'react';
-import { AnchorVanilla as Anchor } from '@xt-pagesource/atomic-react-pattern-lib';
+import { AnchorVanilla as Anchor, sumTest } from '@xt-pagesource/atomic-react-pattern-lib';
```
[the remark this answers is not in the document]
Comment (chandrakailash1993) [reply app/components/templates/HomePage/HomePage.js]: resolved

File: app/components/atoms/Button/tests/__snapshots__/Button.style.test.js.snap (hunk 2 above)
Comment (chandrakailash1993) [reply app/components/atoms/Button/tests/__snapshots__/Button.style.test.js.snap]: Done

File: app/components/atoms/Button/tests/__snapshots__/Button.style.test.js.snap (hunk 3 above)
Comment (chandrakailash1993) [reply app/components/atoms/Button/tests/__snapshots__/Button.style.test.js.snap]: Done

File: app/components/atoms/Image/tests/__snapshots__/Image.style.test.js.snap (hunk 4 above)
Comment (chandrakailash1993) [reply app/components/atoms/Image/tests/__snapshots__/Image.style.test.js.snap]: Done

File: app/components/atoms/ProgressCircle/tests/__snapshots__/ProgressCircle.style.test.js.snap (hunk 5 above)
Comment (chandrakailash1993) [reply app/components/atoms/ProgressCircle/tests/__snapshots__/ProgressCircle.style.test.js.snap]: For few components we have two created two test cases one with StyledComponent and other vanilla

File: app/components/atoms/TextArea/tests/__snapshots__/TextArea.test.js.snap (hunk 6 above)
Comment (chandrakailash1993) [reply app/components/atoms/TextArea/tests/__snapshots__/TextArea.test.js.snap]: removed

File: app/components/molecules/NavBar/tests/__snapshots__/NavBar.test.js.snap (hunk 7 above)
Comment (chandrakailash1993) [reply app/components/molecules/NavBar/tests/__snapshots__/NavBar.test.js.snap]: Done

File: app/components/molecules/TopBanner/tests/__snapshots__/TopBanner.test.js.snap (hunk 8 above)
Comment (chandrakailash1993) [reply app/components/molecules/TopBanner/tests/__snapshots__/TopBanner.test.js.snap]: Done

**Pull request merged**